src/extrainterpreters/piped_interpreter.py

```diff
@@ -41,8 +41,8 @@ class FuncData(StructBase):
 def _dispatcher(pipe, buffer):
     """the core running function in a PipedInterpreter
 
-    This is responsible for watching comunications with the parent
-    interpreter, dispathing execution, and place the return values
+    This is responsible for watching communications with the parent
```

Comment:
as you can see, my code editor does not spell check English - :-) I always have a hard time with doubled consonants.